automation/test.sh

```diff
@@ -28,6 +28,16 @@ readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluste
 
 
 if [ ${CI} == "true" ]; then
+  git remote add upstream https://github.com/kubevirt/kubevirt.git
+  git fetch upstream
+
+  total_lines=$(git diff --name-status upstream/main | wc -l)
+  lines=$(git diff --name-status upstream/main | grep '\.md$' | wc -l)
+  if [ $lines == $total_lines ]; then
```

Comment:
There is an edge case when no lines have changed at all.
Probably should not bother us, but 0 is always a case which does not fit any filter.

Reply:
zero changed lines won't create a pr, hence no tests